mitosheet/src/components/taskpanes/UpdateImports/UpdateImportsPreReplayTaskpane.tsx

```diff
@@ -63,10 +67,12 @@ const getErrorMessage = (
 
     let errorText: string | undefined = undefined;
 
-    if (numImportsInvalidPreReplayThatAreNotUpdated > 0 || numImportsInvalidPostReplay > 0) {
-        errorText = 'Please fix the failed imports below before updating imports';
-    } else if (invalidReplayError !== undefined) {
-        errorText = invalidReplayError;
+    if (!loadingImportDataAndErrors) {
+        if (numImportsInvalidPreReplayThatAreNotUpdated > 0 || numImportsInvalidPostReplay > 0) {
+            errorText = 'Please fix the failed imports below before updating imports';
```

Comment:
Maybe this can say "Please fix the errors below before updating imports". Otherwise, its a bit confusing with fixing imports before updating imports..